sw/device/tests/crypto/ecdsa_p256_verify_functest.c

```diff
@@ -0,0 +1,88 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/crypto/drivers/hmac.h"
+#include "sw/device/lib/crypto/drivers/otbn.h"
+#include "sw/device/lib/crypto/ecdsa_p256/ecdsa_p256.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/check.h"
+#include "sw/device/lib/testing/test_framework/ottf.h"
+#include "sw/device/tests/crypto/ecdsa_p256_verify_testvectors.h"
+
+hmac_error_t compute_digest(size_t msg_len, uint8_t *msg,
+                            ecdsa_p256_message_digest_t *digest) {
+  // Compute the SHA-256 digest using the HMAC device.
+  hmac_sha256_init();
+  hmac_error_t err = hmac_sha256_update(msg, msg_len);
+  if (err != kHmacOk) {
+    return err;
+  }
+  hmac_digest_t act_digest;
```

Comment:
does "act" stand for "actual" here? Not a fan of this contraction... Can you pick a better one?

Reply:
Done, and also updated the existing `ecdsa_p256_functest.c` that I copied this function from so they continue to match.